clients/go/commands/createJob_command.go

```diff
@@ -0,0 +1,144 @@
+package commands
+
+import (
+    "context"
+    "fmt"
+    "github.com/zeebe-io/zeebe/clients/go/pb"
+    "github.com/zeebe-io/zeebe/clients/go/utils"
+    "time"
+)
+
+type DispatchCreateJobCommand interface {
+    Send() (*pb.CreateJobResponse, error)
+}
+
+type CreateJobCommandStep1 interface {
+    JobType(string) CreateJobCommandStep2
+}
+
+type CreateJobCommandStep2 interface {
+    DispatchCreateJobCommand
+
+    SetCustomHeaderFromString(string) (CreateJobCommandStep2, error)
+    SetCustomHeaderFromStringer(fmt.Stringer) (CreateJobCommandStep2, error)
+    SetCustomHeaderFromMap(map[string]interface{}) (CreateJobCommandStep2, error)
+    SetCustomHeaderFromObject(interface{}) (CreateJobCommandStep2, error)
+
+    AddCustomHeader(string, interface{}) CreateJobCommandStep2
+
+    Retries(int32) CreateJobCommandStep2
+
+    PayloadFromString(string) (CreateJobCommandStep2, error)
+    PayloadFromStringer(fmt.Stringer) (CreateJobCommandStep2, error)
+    PayloadFromMap(map[string]interface{}) (CreateJobCommandStep2, error)
+    PayloadFromObject(interface{}) (CreateJobCommandStep2, error)
+}
+
+type CreateJobCommand struct {
+    utils.SerializerMixin
+
+    customHeaders map[string]interface{}
+
+    request       *pb.CreateJobRequest
+    gateway       pb.GatewayClient
+
+}
+
+func (cmd *CreateJobCommand) GetRequest() *pb.CreateJobRequest {
+    return cmd.request
+}
+
+func (cmd *CreateJobCommand) GetCustomHeaders() map[string]interface{} {
```

Comment:
Again normally I don't see getters on the other commands, if it is again for testing than the same applies as with my comment on `GetRequest`.